src/app/loadout-builder/filter/LockedArmor2ModIcon.tsx

```diff
@@ -13,7 +13,7 @@ interface Props {
 
 function LockedArmor2ModIcon({ item, defs, onModClicked }: Props) {
   return (
-    <ClosableContainer onClose={onModClicked}>
+    <ClosableContainer onClose={onModClicked} showCloseIconOnHover>
```

Comment:
We generally spell this out as showCloseOnHover={true}

Reply:
Done